### ISSUE TITLE
Crash report alpha-39: NPE in EditPostActivity.addMediaVisualEditor

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#5412]
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'org.wordpress.android.ui.prefs.SiteSettingsInterface org.wordpress.android.ui.prefs.SiteSettingsInterface.init(boolean)' on a null object reference
       at org.wordpress.android.ui.posts.EditPostActivity.addMediaVisualEditor(EditPostActivity.java:1727)
       at org.wordpress.android.ui.posts.EditPostActivity.addMedia(EditPostActivity.java:1695)
       at org.wordpress.android.ui.posts.EditPostActivity.onPhotoChooserMediaChosen(EditPostActivity.java:595)
       at org.wordpress.android.ui.posts.photochooser.PhotoChooserFragment$4.onItemTapped(PhotoChooserFragment.java:140)
       at org.wordpress.android.ui.posts.photochooser.PhotoChooserAdapter$ThumbnailViewHolder$1.onSingleTapConfirmed(PhotoChooserAdapter.java:288)
       at android.view.GestureDetector$GestureHandler.handleMessage(GestureDetector.java:300)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

**Comment by aforcier** [comment]
I was just opening this. It's also reproducible in 7.0-rc-1.

It seems to have been introduced in https://github.com/wordpress-mobile/WordPress-Android/commit/ba283bea44ef5460e4d917ac2452d39365d91f1f - the return of `init` is `null` for self-hosted sites.

**Comment by aforcier** [comment]
Fixed in #5415.
